### ISSUE TITLE
Netlify hosting

### ISSUE THREAD

**Issue by the-eagle-nest** [nbarkhina/N64Wasm#47]
Is it possible to host this on Netlify?

**Comment by nbarkhina** [comment]
I don't know much about Netlify but as long as it can host and serve up html and javascript files I don't see why not

**Comment by nbarkhina** [comment]
Yeah just drag the "dist" folder in this repo to your Netflify site and it should work

**Comment by the-eagle-nest** [comment]
Thanks, I will try that. I just figured it would need some type of backend server. Or is that only for the cloud part?

**Comment by nbarkhina** [comment]
Yeah that's only if you want cloud saves. Otherwise it's just static files no server code needed.
